src/main/kotlin/no/nav/tilleggsstonader/sak/statistikk/behandling/dto/DvhEnums.kt

```diff
@@ -6,7 +6,6 @@ enum class Hendelse {
     VENTER,
     VEDTATT,
     BESLUTTET,
-    HENLAGT,
```

Comment:
Er dette en status de ønsker? Hvis ikke så tenker jeg vi kan bruke FERDIG for henlagte behandlinger. Då får vi allt av statuser som trengs for FERDIG-hendelser, eks `ferdigBehandletTid`

Reply:
De har ikke uttrykt noe spesifikt om hva de ønsker her, men på et generelt grunnlag vil de ha "mest mulig". Så blir det opp til oss å bestemme hva som blir mest mulig informativt.

På EF ser jeg de bruker FERDIG-statusen for henlagte saker, så det bør vi nok også gjøre ja. Good catch.  

Hva tror du, @janeirik ?

Reply:
Stemmer for at me bruker ferdig også for henlagt.